src/data/JsonResume.ts

```diff
@@ -0,0 +1,399 @@
+  image?: URL;
+
+  /**
+   * Email address to contact the individual.
+   *
+   * For example: thomas@jsonresume.org
+   */
+  email?: string;
+
+  /**
+   * Phone numbers, can be specified in any format.
+   *
+   * For example: 712-117-2923 or +44 7267038141
+   */
+  phone?: string;
+
+  /**
+   * URL to the individuals website.
+   *
+   * For example: a portfolio or personal homepage
+   */
+  url?: URL;
+
+  /** A short bio about the individual. */
+  summary?: string;
+
+  /** Where the individual is located. */
+  location?: Location;
+
+  /** An array of social media networks the individual is on. */
+  profiles?: Profile[];
+}
+
+/** The address to a physical location. */
+interface Location {
+  /**
+   * To add multiple address lines, use \n.
+   *
+   * For example: 1234 Glücklichkeit Straße\nHinterhaus 5. Etage li.
+   */
+  address?: string;
+
+  /** Postal code for the location. */
+  postalCode?: string;
+
+  city?: string;
+
+  /**
+   * ISO-3166-1 ALPHA-2 code for the country.
+   *
+   * For example: US, AU, IN
+   */
+  countryCode?: string;
+
+  /** The general region where you live. Can be a state, county, or province. */
+  region?: string;
+}
+
+/** Social media profile. */
+interface Profile {
+  /**
+   * The name of the network or platform.
+   *
+   * For example: Facebook or Twitter
+   */
+  network?: string;
+
+  /**
+   * Username on the platform.
+   *
+   * For example: elonmusk
+   */
+  username?: string;
+
+  /**
+   * URL to the profile on the platform.
+   *
+   * For example: https://twitter.com/elonmusk
+   * */
+  url?: URL;
+}
+
+/** A work entry on the résumé. */
+interface Work extends Occupation {
+  /**
+   * The name of the organization.
+   *
+   * For example: Facebook
+   */
+  name?: string;
+
+  /** e.g. Menlo Park, CA */
+  location?: string;
+
+  /** e.g. Social Media Company */
+  description?: string;
+
+  /** e.g. Software Engineer */
+  position?: string;
+
+  /** e.g. https://facebook.com */
+  url?: URL;
+
+  /** An overview of the responsibilities at the organization. */
+  summary?: string;
+
+  /** Accomplishments achieved during the position. */
+  highlights?: string[];
+}
+
+interface Volunteer extends Occupation {
+  /** e.g. Facebook */
+  organization?: string;
+
+  /** e.g. Software Engineer */
+  position?: string;
+
+  /** e.g. https://facebook.example.com */
+  url?: URL;
+
+  /** Give an overview of your responsibilities at the company */
+  summary?: string;
+
+  /** Specify accomplishments and achievements */
+  highlights?: string[];
+}
+
+interface Education extends Occupation {
+  /** e.g. Massachusetts Institute of Technology */
+  institution?: string;
+
+  /** e.g. https://facebook.example.com */
+  url?: URL;
+
+  /** e.g. Arts */
+  area?: string;
+
+  /** e.g. Bachelor */
+  studyType?: string;
+
+  /** grade point average, e.g. 3.67/4.0 */
+  score?: string;
+
+  /** List notable courses/subjects */
+  courses?: string[];
+}
+
+interface Award {
```

Comment:
I do have a lot of interfaces that I don't need in this file, since I don't use them at all. However, it would be nice to keep them, since it's possible a user may have those fields in their JSON Resume file.